packages/core/content-manager/server/routes/admin.js

```diff
@@ -182,6 +182,21 @@ module.exports = {
         ],
       },
     },
+    {
+      method: 'GET',
+      path: '/single-types/:model/:id/actions/numberOfDraftRelations',
+      handler: 'single-types.getNumberOfDraftRelations',
+      config: {
+        middlewares: [routing],
+        policies: [
+          'admin::isAuthenticatedAdmin',
+          {
+            name: 'plugin::content-manager.hasPermissions',
+            config: { actions: ['plugin::content-manager.explorer.read'] },
+          },
+        ],
+      },
+    },
```

Comment:
The FE has no access to a single types id from the EditView, therefore I think this route will need to run without access to the `id`
```suggestion
    {
      method: 'GET',
      path: '/single-types/:model/actions/numberOfDraftRelations',
      handler: 'single-types.getNumberOfDraftRelations',
      config: {
        middlewares: [routing],
        policies: [
          'admin::isAuthenticatedAdmin',
          {
            name: 'plugin::content-manager.hasPermissions',
            config: { actions: ['plugin::content-manager.explorer.read'] },
          },
        ],
      },
    },
```